api/helpers/game-states/moves/stalemate-reject/execute.js

```diff
@@ -42,11 +42,7 @@ module.exports = {
       ...result,
       ...requestedMove,
       playedBy,
-      phase,
-      playedCard: null,
-      targetCard: null,
-      discardedCards: [],
-      resolved: null,
```

Comment:
Removing these so that they can keep their same value from the previous move. This lets the check on whether a card is frozen read from these properties to identify if they should still be frozen after a resolve 9